Apsitvarkom.UnitTests/Models/Validation/PollutedLocationValidationTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Apsitvarkom.Models;
+using FluentValidation;
+
+namespace Apsitvarkom.UnitTests.Models.Validation
+{
+    internal class PollutedLocationValidationTests
+    {
+        private static readonly IValidator<PollutedLocation> Validator = new PollutedLocationValidator();
+        private static readonly Guid dummyGuid = Guid.NewGuid();
+
+        private static readonly PollutedLocation[] ValidPollutionLocations =
+        {
+            new()
+            {
+                Id = dummyGuid,
+                Location = new Location
+                {
+                    Latitude = 35.929673,
+                    Longitude = -78.948237,
+                },
+                Notes = "I love writting tests",
```

Comment:
```suggestion
                Notes = "I love writing tests",
```
A minor typo that doesn't affect anything, but as changes will be needed, we can fix this too. :D